includes/Core/Util/Auto_Updates.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Auto_Updates
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Utility class for auto-updates settings.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Auto_Updates {
+	/**
+	 * Checks whether plugin auto-updates are enabled for the site.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool `false` if auto-updates are disabled, `true` otherwise.
+	 */
+	public static function is_plugin_autoupdates_enabled() {
+		if ( false === self::is_sitekit_autoupdates_forced() ) {
+			return false;
+		}
```

Comment:
I'm a bit unclear why we're checking for Site Kit's status specifically here when this is about plugin auto-update status in general. I know for the _existing_ use-case it's functionally the case that we won't show auto-update UI if the user can't set Site Kit to auto-update, but what if they have Site Kit forced to not auto-update but plugins in general _can_ auto-update? We might eventually want to show a warning to users about that, and this check would prevent us from deducing that state, I think 🤔

Reply:
I understand, so to solve this, we would have to add one additional attribute and selector to the `core/site` state that includes the forced state and then add the logic to the `EnableAutoUpdateBannerNotification` banner.

However the problem with that is, the `auto_update_plugin` filter which the `wp_is_auto_update_forced_for_item` is a wrapper for, is often used to generally disable all plugin updates, althought the `plugins_auto_update_enabled` filter should be used for that. This is the case for many "disable plugin updates"-Plugins from the WP plugin directory and many "how to disable auto updates" how-to's. Even the WordPress Docs are not clear about this and recommend to use this filter to disable plugin updates generally, [see here](https://developer.wordpress.org/reference/hooks/auto_update_type/#:~:text=add%20the%20code%20snippet%20to%20disable%20automatic%20WordPress%20plugin%20updates.).
Therefore there is no way for us to verify if this "forced" functionality is used to disable auto updates only for Site Kit or for all plugins.

That's why I added this `is_sitekit_autoupdates_forced()` check here, as thats whats needed/works by now.

But I also understand your concerns, so I'm unsure what to do. What do you think about this?

Reply:
```php
 $data['updatePluginCapacity']      = current_user_can( 'update_plugins' ) && Auto_Updates::AUTO_UPDATE_NOT_FORCED === Auto_Updates::sitekit_forced_autoupdates_status(); 

´´´

Reply:
Thanks so much for explaining this one in detail. Let's go with your approach, including renaming the capability.

Can you add a code comment explaining this (and/or linking to this discussion) so future developers can make sense of this? It's a bit complicated! 😅 

But I think your idea here is good. Thanks! 👍🏻